person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_accessor :name, :age
+
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    if of_age? || @parent_permission
+      true
+    else
+      false
+    end
```

Comment:
- [OPTIONAL] This is a good check. But let's keep in mind that ruby is a flexible language and provide a lot of shorter syntax. 

See example:
```ruby
is_of_age?() || @parent_permission
```